documentation/asciidoc/topics/proc_disabling_gossip_router_cross_site.adoc

```diff
@@ -0,0 +1,46 @@
+[id='disabling-gossip-router-cross-site_{context}']
+= Disabling local Gossip router and service
+
+[role="_abstract"]
+The {ispn_operator} starts a Gossip router on each site, but you only need a single Gossip router to manage traffic between the {brandname} cluster members.
+You can disable the additional Gossip routers to save resources.
+
+.Prerequisites
+
+* Have multiple working Gossip routers.
++
+You can have an external Gossip router that is not managed by the {ispn_operator}.
```

Comment:
This isn't correct. The Gossip Router will be managed by the Operator, however it will only be enabled on one of the sites, therefore it's only managed by the Operator on the enabled site.